resilience4j-retry/src/main/java/io/github/resilience4j/retry/Retry.java

```diff
@@ -350,6 +376,13 @@ static Runnable decorateRunnable(Retry retry, Runnable runnable) {
 	 */
 	RetryConfig getRetryConfig();
 
+	/**
+	 * Returns an unmodifiable map with tags assigned to this Retry.
+	 *
+	 * @return the tags assigned to this Retry in an unmodifiable map
+	 */
+	Map<String, String> getTags();
```

Comment:
Could we return an immutable Map from Vavr instead?

Comparable to `Seq<CircuitBreaker> getAllCircuitBreakers();`

Reply:
You mean `io.vavr.collection.Map`? Sounds good to me. Am I right to assume you want to use that everywhere? For example also in the `static Retry of(String name, RetryConfig retryConfig, Map<String, String> tags)` methods and such?

Reply:
Yes